instrumentation/opentelemetry_ecto/mix.exs

```diff
@@ -6,7 +6,7 @@ defmodule OpentelemetryEcto.MixProject do
       app: :opentelemetry_ecto,
       description: description(),
       version: "1.1.1",
-      elixir: "~> 1.10",
+      elixir: "~> 1.14",
```

Comment:
We should always leave this as low as the library will allow. Unless we require a feature from a newer version, we should go as low as possible.

This note applies to all the libraries.